### ISSUE TITLE
Unwanted feature values retrieved when no feature flags set within Azure App Configuration

### ISSUE THREAD

**Issue by JakeRoper4** [microsoft/FeatureManagement-Dotnet#91]
I've been working with your library for some time now, and have got it working so that a list of feature flags is passed through an API of my own creation, via Azure App Configuration and the use of this package. However, is there a way to limit what feature store `featureManager.GetFeatureNamesAsync()` has access to? I only want to poll my AAC connection with this method, but currently, if no features are found within my instance of Azure App Configuration, it will fall back to using what appears to be environment variables of some kind?

TL:DR - Is there a way to limit the featureManager to access only one store/provider?

**Comment by jimmyca15** [comment]
It is possible to pass in an instance of `IConfiguration` that is separate from the apps default `IConfiguration` instance that contains non feature related settings. This requires building a separate `IConfiguration` instance.

I would say if you added an empty `FeatureManagement` section that should work to hide the unwanted values. It seems like a bug and I'll look into fixing it. It sounds like that might be the best approach here.

Another option is to provide a setting in the feature manager to disable the functionality that falls back to the root of the `IConfiguration` instance. Although, I'd like to just use the empty FeatureManagement section approach.

**Comment by JakeRoper4** [comment]
Thanks for the reply @jimmyca15 - I do like the approach of leaving an empty section within AppSettings to fall back on, though I'm unsure currently whether it's something that I've done wrong, or something that perhaps isn't working correctly with the fallback mechanism. 

Let me know if you need any more information from me regarding this, as I'm quite keen to avoid displaying unwanted configuration to callers of my app/api. 

Just as a lil side note, below is the empty section of `FeatureManagement` in my appSettings.json file.

```
    "FeatureManagement": {
    },
```

**Comment by JakeRoper4** [comment]
Just to make you aware. I had passed in the whole configuration instance into the `AddFeatureManagement` service before, which led me to the above error. I had looked at some tutorials on how other users have integrated this functionality, and this led me to just passing in the blank `FeatureManagement` section within configuration as below:

`services.AddFeatureManagement(Configuration.GetSection("FeatureManagement"))`

After building the solution with the above line in, and removing all features within my feature manager, I had the expected result of an empty list within the response of my API call. Good stuff! Thank you for replying once again, but I'll close this as resolved.
